projects/kernel-config/Makefile

```diff
@@ -11,7 +11,7 @@
 # Git tree hash of this directory. Override to force build
 HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 # Name on Hub
-IMAGE:=kernel
+IMAGE?=kernel
```

Comment:
@tych0 just created and add you to `linuxkit/kernel-probational` https://hub.docker.com/r/linuxkit/kernel-probational

`linuxkit/kernel-config` already exists and hasn't been pushed for a while, added you to that as well:  https://hub.docker.com/r/linuxkit/kernel-config